Dynatrace/dynatrace-operator
# Use latest version endpoint for agent download

# Description
Custom version defined on the Tenant was ignored in favor of latest available version. 

## How can this be tested?
Change what oneagent version is used from latest on the Tenant and check correct version of code modules is downloaded.

## Checklist
